scripts/create-example-data.js

```diff
@@ -110,7 +109,7 @@ async function createExampleData(secret) {
       .then(createThen(`Alice Todos`))
       .catch(createCatch(`Alice Todos`))
 
-    return Promise.all([aliceTodoPromise, nancyTodoPromise])
+    return await Promise.all([aliceTodoPromise, nancyTodoPromise])
```

Comment:
This was another important change - the code would delete the test key before these actions completed, causing them to fail.